sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/ChatViewModel.kt

```diff
@@ -141,6 +136,24 @@ abstract class ChatViewModel<ARGS: NavArgs>(
                     )
                 )
 
+                if (chat?.status?.isPending() == true) {
+                    submitSideEffect(
+                        ChatSideEffect.Notify(
+                            app.getString(R.string.waiting_for_admin_approval),
+                            false
+                        )
+                    )
+                }
+
+                if (chat?.status?.isApproved() != true) {
+                    footerViewStateContainer.updateViewState(
+                        FooterViewState.Disabled
+                    )
+                } else {
+                    footerViewStateContainer.updateViewState(
+                        FooterViewState.Default
+                    )
+                }
```

Comment:
This will update the `FooterViewState` on every change to the Chat Table with no consideration to the current FooterViewState (if they menu is open and user is sending an image attachment, for example).